lang/cs/Org.Apache.REEF.IO/FileSystem/Hadoop/HadoopFileSystem.cs

```diff
@@ -47,11 +47,20 @@ private HadoopFileSystem(HdfsCommandRunner commandRunner)
         }
 
         /// <summary>
-        /// The Prefix used for URIs on this FileSystem.
+        /// CReate Uri from a given file name
```

Comment:
Please fix typos. Also, the assumption here is that the path given is a relative path on the FileSystem, right? What if it is already an Uri that points to another filesystem?